deploy/charts/emqx/templates/StatefulSet.yaml

```diff
@@ -47,6 +47,10 @@ spec:
         version: {{ .Chart.AppVersion }}
         app.kubernetes.io/name: {{ include "emqx.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
+      {{- if .Values.recreatePods }}
+      annotations:
+        recreated-pods-at: {{ now | quote }}
```

Comment:
make sense to annotate with configmap hash instead ?
e.g.
`checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}`

ref: https://v2.helm.sh/docs/developing_charts/#automatically-roll-deployments-when-configmaps-or-secrets-change

Reply:
@zmstone I fixed this and additionally also took care of the warnings for Kubernetes `>=1.17`.